PactSafe iOS Demo/Controllers/SignUpViewController.swift

```diff
@@ -6,145 +6,146 @@
 //  Copyright © 2019 Tim Morse . All rights reserved.
 //
 
-import UIKit
-import SafariServices
 import FirebaseAuth
 import PactSafe
+import SafariServices
+import UIKit
 
 class SignUpViewController: UIViewController {
-    
-    //MARK: IBOutlets
-    @IBOutlet weak var firstName: UITextField!
-    @IBOutlet weak var lastName: UITextField!
-    @IBOutlet weak var emailAdress: UITextField!
-    @IBOutlet weak var password: UITextField!
-    @IBOutlet weak var reenterPassword: UITextField!
-    @IBOutlet weak var submitButton: PSSubmitButton!
-    @IBOutlet weak var pactSafeClickWrap: PSClickWrapView!
-    
-    // MARK: - Properties
-    
-    /// The shared instance of the PactSafe app.
-    private let ps = PSApp.shared
-
-    // MARK: -  Methods
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        // Optionally use a UITextView Delegate to open linked contracts in SFSafariViewController
-        pactSafeClickWrap.textView.delegate = self
-        
-        // Set up and load contracts into clickwrap
-        pactSafeClickWrap.loadContracts(withGroupKey: "example-mobile-app-group")
-        
-        // Only enable the submit button if checkbox is checked
-        pactSafeClickWrap.checkbox.valueChanged = { (isChecked) in
-            // If isChecked is true, enable submit button
-            self.configureSubmitButtonState(isChecked)
-        }
-        
-        submitButton.setBackgroundColor(UIColor.lightGray, for: .disabled)
-        
-        // Configure the default button state (default is false)
-        configureSubmitButtonState()
+  // MARK: IBOutlets
+
+  @IBOutlet var firstName: UITextField!
+  @IBOutlet var lastName: UITextField!
+  @IBOutlet var emailAdress: UITextField!
+  @IBOutlet var password: UITextField!
+  @IBOutlet var reenterPassword: UITextField!
+  @IBOutlet var submitButton: PSSubmitButton!
+  @IBOutlet var pactSafeClickWrap: PSClickWrapView!
+
+  // MARK: - Properties
+
+  /// The shared instance.
+  private let ps = PSApp.shared
+
+  // MARK: -  Methods
```

Comment:
oh hi Mark

What are these?

Reply:
Standard way of adding points in code that can be easily navigated to within Xcode! Also helpful with the "minimap" feature in Xcode as well for finding groups of code.